test/python/test_tf2ngraph_shape_hints.py

```diff
@@ -141,6 +136,9 @@ class Testtf2ngraphShapehints(NgraphTest):
     @pytest.mark.skipif(
         not ngraph_bridge.is_grappler_enabled(),
         reason="Requires grappler build for tf2ngraph and AOT")
+    @pytest.mark.skipif(
+        NGTF_BACKEND != "INTERPRETER",
+        reason="Only INTERPRETER backend supports precompilation")
```

Comment:
The Interpreter pipeline does not use grappler so it will be skipped on that pipeline and it will be skipped on grappler pipeline because of setting NGRAPH_TF_BACKEND=CPU

Reply:
I checked and this test was getting skipped. I fixed it here #583